src/components/sidebar/__internal__/sidebar-header/sidebar-header.style.js

```diff
@@ -1,21 +1,24 @@
 import styled from "styled-components";
-import baseTheme from "../../../style/themes/base";
+import { padding } from "styled-system";
+import baseTheme from "../../../../style/themes/base";
 
 const SidebarHeaderStyle = styled.div`
+  ${padding};
```

Comment:
I can't find where these padding props are passed to this component. Looks like they are only defined in `defaultProps`.

Reply:
Yeah you need to add the padding props into `SidebarHeader` - similar to how the margin props are added to https://github.com/Sage/carbon/blob/master/src/__experimental__/components/radio-button/radio-button.component.js